src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/IsolatedStorageFile.cs

```diff
@@ -568,33 +568,25 @@ private static IsolatedStorageFile GetStore(IsolatedStorageScope scope)
         public static IsolatedStorageFile GetStore(IsolatedStorageScope scope, Type applicationEvidenceType)
         {
             // Scope MUST be Application
-
-            // https://github.com/dotnet/corefx/issues/10935
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(SR.PlatformNotSupported_CAS); // https://github.com/dotnet/corefx/issues/10935
```

Comment:
What is the applicationEvidenceType used for in desktop?  If it's purely about validating CAS, should this instead just delegate to GetStore(scope) rather than throwing?